src/waypoint.jsx

```diff
@@ -333,13 +352,18 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.children) {
+      return <div ref={this.refElement}>{this.props.children}</div>;
```

Comment:
Ah, nice. If that works that sounds good to me. I don't think it would allow multi-element children because there would be no single containing element in the WaypointRefContainer.